smart-contracts/contracts/OracleStorage.sol

```diff
@@ -31,11 +31,6 @@ contract OracleStorage {
     */
     mapping(uint256 => mapping(address => bool)) private hasMadeClaim;
 
-    /**
-    * @dev {DEPRECATED}
-    */
-    mapping(uint256 => bool) private prophecyRedeemed;
```

Comment:
Was this data structure not used in the current mainnet deployment of peggy?

Reply:
It wasn't.
It's not in the `develop` smart contracts, nor in the deployments folder jsons.